pretrain.py

```diff
@@ -0,0 +1,84 @@
+import torch
+
+from models.BiDirectionalTransformer import BiDirectionalTransformer
+from utils.maskgenerator import BatchJointMaskGenerator
+from utils.dataset import prepare_dataset
+from utils.types import ModelEnum, TargetEnum
+
+
+def main(model_name: str = None, target_type: str = None) -> None:
```

Comment:
Is there anyway we can merge pretrain.py into the current main.py? It seems pretty similar to me

Reply:
Will change